packages/aws-rfdk/lib/core/scripts/bash/installLustreClient.sh

```diff
@@ -0,0 +1,264 @@
+    # For example, see announcement for CentOS 7 (1804): https://lists.centos.org/pipermail/centos-announce/2018-May/022829.html
+    #
+    # The major and minor versions of CentOS releases match that of RHEL,
+    # so we can just extract those bits and treat them as the RHEL version
+    rhel_version=$(awk '{print $4}' /etc/centos-release | awk -F "." '{print $1"."$2}')
+  fi
+
+  case $rhel_version in
+    # RHEL 7.5 and 7.6
+    # ----------------
+    7.5|7.6)
+      case $kernel_version in
+        ${rhel_kernel_versions[7.5]}) lustre_version="2.10.5";;
+        ${rhel_kernel_versions[7.6]}) lustre_version="2.10.8";;
+        *) echo "ERROR: Unsupported kernel version $kernel_version for RHEL $rhel_version"; exit 1;;
+      esac
+      sudo yum -y install "https://downloads.whamcloud.com/public/lustre/lustre-${lustre_version}/el7/client/RPMS/x86_64/kmod-lustre-client-${lustre_version}-1.el7.x86_64.rpm"
+      sudo yum -y install "https://downloads.whamcloud.com/public/lustre/lustre-${lustre_version}/el7/client/RPMS/x86_64/lustre-client-${lustre_version}-1.el7.x86_64.rpm"
+    ;;
+
+    # RHEL 7.7, 7.8, and 7.9
+    # ----------------------
+    7.7|7.8|7.9)
+      architecture=$(uname -m)
+      case $architecture in
+        x86_64)
+          distro_name="el"
+          case $kernel_version in
+            ${rhel_kernel_versions[7.7]}) replace_ver="7.7";;
+            ${rhel_kernel_versions[7.8]}) replace_ver="7.8";;
+            ${rhel_kernel_versions[7.9]});; # Do nothing
+            *) echo "ERROR: Unsupported kernel version $kernel_version for RHEL $rhel_version"; exit 1;;
+          esac
+        ;;
+        aarch64)
+          distro_name="centos"
+          case $kernel_version in
+            # RHEL 7.8 for ARM-based AWS-Graviton powered instances have the 8.1 kernel version (see AWS docs)
+            ${rhel_kernel_versions[8.1]}) replace_ver="7.8";;
+            ${rhel_kernel_versions[8.2]});; # Do nothing
+            *) echo "ERROR: Unsupported kernel version $kernel_version for RHEL $rhel_version"; exit 1;;
+          esac
+        ;;
+        *) echo "ERROR: Unrecognized architecture: $architecture"; exit 1;;
+      esac
+
+      curl https://fsx-lustre-client-repo-public-keys.s3.amazonaws.com/fsx-rpm-public-key.asc -o /tmp/fsx-rpm-public-key.asc
+      sudo rpm --import /tmp/fsx-rpm-public-key.asc
+      sudo curl "https://fsx-lustre-client-repo.s3.amazonaws.com/${distro_name}/7/fsx-lustre-client.repo" -o /etc/yum.repos.d/aws-fsx.repo
+      if [[ -n "${replace_ver+x}" ]]; then
+        sudo sed -i "s#7#${replace_ver}#" /etc/yum.repos.d/aws-fsx.repo
+      fi
+
+      sudo yum clean all
+      sudo yum install -y kmod-lustre-client lustre-client
+    ;;
+
+    # RHEL 8.2 or newer
+    # -----------------
+    8.2|8.[1-9]*([0-9]))
+      case $kernel_version in
+        ${rhel_kernel_versions[8.2]}) replace_ver="8.2";;
+        ${rhel_kernel_versions[8.3]}  replace_ver="8.3";;
```

Comment:
Ah yeah I missed that, added in an entry for 8.3. I went ahead and added an entry for 8.4, but we technically don't need it yet since it's the latest release, so we don't have to download a specific version yet.

Unfortunately, yes. We will need to manually update this whenever a new version of RHEL is released, assuming the FSx Lustre docs also update the [instructions](https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html#lustre-client-rhel) with the new kernel versions added.